ogcore/tax.py

```diff
@@ -43,9 +44,8 @@ def MTR_wealth(b, h_wealth, m_wealth, p_wealth):
     Calculates the marginal tax rate on wealth from the wealth tax.
 
     .. math::
-        \frac{\partial T_{j,s,t}^{w}}{\partial b_{j,s,t}} =
-        \frac{h^{w}p_{w}b_{j,s,t}}{(b_{j,s,t}h^{w}+m^{w})}\left[2 -
-        \frac{h^{w}p_{w}b_{j,s,t}}{(b_{j,s,t}h^{w}+m^{w})}\right]
+        \tau^{mtrw}_{t} = \tau^{etr,w}_{t}
```

Comment:
@jdebacker. In the documentation, I deliberately made this the notational choice (see `equilibrium.md`, `government.md`, `households.md`, and `stationarization.md`). Because `tau^{etr,xy}` and `tau^{etr,w}` are shown in the same equation (see equation 57), I put a comma between "etr" and the type of effective tax rate. However, I decided not to change the `mtrx` and `mtry` specifications that don't have any commas. And I decided to just show `mtrw` to be parallel. If you want me to update this, I'll have to go through all the documentation and code and update all the instances. I recommend we do not do this.

Reply:
Ah - makes sense.